perception/radar_fusion_to_detected_object/src/radar_object_fusion_to_detected_object_node/radar_object_fusion_to_detected_object_node.cpp

```diff
@@ -61,31 +61,29 @@ RadarObjectFusionToDetectedObjectNode::RadarObjectFusionToDetectedObjectNode(
     std::bind(&RadarObjectFusionToDetectedObjectNode::onSetParam, this, _1));
 
   // Node Parameter
-  node_param_.update_rate_hz = declare_parameter<double>("node_params.update_rate_hz", 10.0);
+  node_param_.update_rate_hz = declare_parameter<double>("node_params.update_rate_hz");
 
   // Core Parameter
-  core_param_.bounding_box_margin =
-    declare_parameter<double>("core_params.bounding_box_margin", 0.5);
+  core_param_.bounding_box_margin = declare_parameter<double>("core_params.bounding_box_margin");
   core_param_.split_threshold_velocity =
-    declare_parameter<double>("core_params.split_threshold_velocity", 0.0);
-  core_param_.threshold_yaw_diff =
-    declare_parameter<double>("core_params.threshold_yaw_diff", 0.35);
+    declare_parameter<double>("core_params.split_threshold_velocity");
+  core_param_.threshold_yaw_diff = declare_parameter<double>("core_params.threshold_yaw_diff");
   core_param_.velocity_weight_min_distance =
-    declare_parameter<double>("core_params.velocity_weight_min_distance", 1.0);
+    declare_parameter<double>("core_params.velocity_weight_min_distance");
   core_param_.velocity_weight_average =
-    declare_parameter<double>("core_params.velocity_weight_average", 0.0);
+    declare_parameter<double>("core_params.velocity_weight_average");
   core_param_.velocity_weight_median =
-    declare_parameter<double>("core_params.velocity_weight_median", 0.0);
+    declare_parameter<double>("core_params.velocity_weight_median");
   core_param_.velocity_weight_target_value_average =
-    declare_parameter<double>("core_params.velocity_weight_target_value_average", 0.0);
+    declare_parameter<double>("core_params.velocity_weight_target_value_average");
   core_param_.velocity_weight_target_value_top =
-    declare_parameter<double>("core_params.velocity_weight_target_value_top", 1.0);
+    declare_parameter<double>("core_params.velocity_weight_target_value_top");
   core_param_.convert_doppler_to_twist =
-    declare_parameter<bool>("core_params.convert_doppler_to_twist", false);
+    declare_parameter<bool>("core_params.convert_doppler_to_twist");
   core_param_.threshold_probability =
-    declare_parameter<float>("core_params.threshold_probability", 0.0);
```

Comment:
[MUST] Probability value is defined by float in Autoware message. Please revert to float.

Reply:
Hi, it was "float" in my previous commit, but @ambroise-arm asked me to change to double since the new Declare Parameter Function. ( see in https://github.com/autowarefoundation/autoware.universe/pull/4537#discussion_r1297064534) 

Please let me know which to follow. Thanks!

Reply:
I am not sure how the change from float to double relates to the pre-commit.ci failure. It is not a build job but a linting job, and the failures it reports look unrelated to the changes in this PR.
Let's see what the `build-and-test*` jobs result in for building the code.

Reply:
There are no pre-commit.ci failures in other PR. So I guess build-and-test* jobs don't have problem and your change cause pre-commit.ci failures.
I recommend that you construct local environment and debug it.

Reply:
Hi @scepter914,
Finally, pre-commit.ci check has passed.

Reply:
@PhoebeWu21 
[imo] Since `core_param_.threshold_probability` is [defined](https://github.com/PhoebeWu21/autoware.universe/blob/refactor-node-config-radar_fusion_to_detected_object/perception/radar_fusion_to_detected_object/include/radar_fusion_to_detected_object.hpp#L66) as float, this is implicit type conversion here. 
Is this meant?